### PR TITLE
QUEST200 - # 9070 - 9090 + Accessory Shop update.

### PR DESCRIPTION
**Pull request by vtange**
Please review especially the lines 9080 - 9090. I don't know any of the names/context but tried my best to fill things in.

Accessory Shop lines updated to what I made back in #936 (quest100 #2277)

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -9063,27 +9063,27 @@ QUEST_LV_0200_20150908_009066	You've been pricked by the thorns of bushes!
...
+QUEST_LV_0200_20150918_009084	You've come to want something, huh? {nl} An interesting life really depends on reckless challenges, huh?
+QUEST_LV_0200_20150918_009085	$Well, well. What have I been doing? {nl} Oh, my Lylia...{nl}
+QUEST_LV_0200_20150918_009086	$Her music box and necklace was snatched by the subordinates, and infused with Siktam's Curse. It's as if they are trying to mock Maven's teachings.
+QUEST_LV_0200_20150918_009087	Hold on a sec, my head's ringing again from that curse. {nl} There's still so much to say... I don't know if I will be able to handle it at that moment.
+QUEST_LV_0200_20150918_009088	Thank you very much. It's really gotten better, huh. {nl} The music box.. My Lylia.. She may no longer be in this world. {nl}
+QUEST_LV_0200_20150918_009089	If you run into Naktis' possessions, please be sure to carefully inspect it for her sake. {nl} Our symbol of Love, who drank in that demonic curse... We are to give her a thankful welcome with revenge.
+QUEST_LV_0200_20150918_009090	That is a necklace blessed by the cathedral, so it must be of high utility. {nl} To me that necklace no longer holds any meaning.
```
Comment (ghost) [inline QUEST_LV_0200.tsv]: Add a comma after "me".

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -9063,27 +9063,27 @@ QUEST_LV_0200_20150908_009066	You've been pricked by the thorns of bushes!
...
-QUEST_LV_0200_20150918_009084	
-QUEST_LV_0200_20150918_009085	
-QUEST_LV_0200_20150918_009086	
-QUEST_LV_0200_20150918_009087	
-QUEST_LV_0200_20150918_009088	
-QUEST_LV_0200_20150918_009089	
-QUEST_LV_0200_20150918_009090	
+QUEST_LV_0200_20150908_009070	You've dug the gold plate well!
+QUEST_LV_0200_20150918_009071	$I've been able to regain some of my power thanks to you, but this is only the beginning  -- {nl} I'll help protect you now by luring the demons with the fake revelation at the tower. {nl}
+QUEST_LV_0200_20150918_009072	$Please take Laima's revelation to the Cathedral. Believe in yourself and never hold any doubts. Victory is certainly yours in the end.
```
Comment (ghost) [inline QUEST_LV_0200.tsv]: "Victory is certainly yours in the end." -> "Victory will certainly be yours in the end."

?

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -9063,27 +9063,27 @@ QUEST_LV_0200_20150908_009066	You've been pricked by the thorns of bushes!
...
+QUEST_LV_0200_20150918_009076	$Is that the Jar of Fire? I know about it. However, it's no good in that condition. Maybe the blue tar found in Saknies Plains could fix it...
+QUEST_LV_0200_20150918_009077	$If it's not necessary then I also don't have a choice. {nl} I am a merchant, not a volunteer.
+QUEST_LV_0200_20150918_009078	$Oh, thank you very much. {nl} The Jar of Fire was once with a monster called Mothstem. I've heard it would never let go of it.
+QUEST_LV_0200_20150918_009079	$Um, I heard the general goods trader saw it recently. We should go ask him about it.
+QUEST_LV_0200_20150918_009080	What a tragedy. It's regrettable. {nl} For such hard feelings to be placed upon a merchant who only knows about money like me...
+QUEST_LV_0200_20150918_009081	However, if I have the people who depend on me wait, there will be more with broken equipment.
+QUEST_LV_0200_20150918_009082	We don't ask for a lot. {nl} You can see this in the signs the pilgrims make along Starving Demon's Way.
+QUEST_LV_0200_20150918_009083	If no then we don't have a choice. {nl} If we can, we will mutually profit.
```
Comment (ghost) [inline QUEST_LV_0200.tsv]: Add a comma after "no".

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -9063,27 +9063,27 @@ QUEST_LV_0200_20150908_009066	You've been pricked by the thorns of bushes!
...
+QUEST_LV_0200_20150908_009070	You've dug the gold plate well!
+QUEST_LV_0200_20150918_009071	$I've been able to regain some of my power thanks to you, but this is only the beginning  -- {nl} I'll help protect you now by luring the demons with the fake revelation at the tower. {nl}
+QUEST_LV_0200_20150918_009072	$Please take Laima's revelation to the Cathedral. Believe in yourself and never hold any doubts. Victory is certainly yours in the end.
+QUEST_LV_0200_20150918_009073	$Please come. {nl} We're short on many things, so we have to make use of every bit we've got.
+QUEST_LV_0200_20150918_009074	$A Mothstem? The Fedimian Watch said they saw one just outside the outer walls. You should find the Pardoner Master. Do you still not know?
+QUEST_LV_0200_20150918_009075	$Hurry back to the Pardoner Master. She has been looking for that Jar of Fire for a long time.
+QUEST_LV_0200_20150918_009076	$Is that the Jar of Fire? I know about it. However, it's no good in that condition. Maybe the blue tar found in Saknies Plains could fix it...
+QUEST_LV_0200_20150918_009077	$If it's not necessary then I also don't have a choice. {nl} I am a merchant, not a volunteer.
```
Comment (ghost) [inline QUEST_LV_0200.tsv]: Add a comma after "necessary".

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile: 

(edited some lines from 9080 - 9090 :wink: )

**Pull request merged**